### ISSUE TITLE
Red and blue color channel switched?

### ISSUE THREAD

**Issue by s9w** [wolfpld/tracy#184]
When I specify blue, it gets printed red. Green stays the same. A look at the [code](https://github.com/wolfpld/tracy/blob/57d636f2432048d6b209568afb3b8ec90d0aafae/client/TracyProfiler.cpp#L3499) says:

```
MemWrite( &item.srcloc.r, uint8_t( ( srcloc->color       ) & 0xFF ) );
MemWrite( &item.srcloc.g, uint8_t( ( srcloc->color >> 8  ) & 0xFF ) );
MemWrite( &item.srcloc.b, uint8_t( ( srcloc->color >> 16 ) & 0xFF ) );
```

I'm not doing bitmasking often, but that seems to confirm my suspicion.

**Comment by wolfpld** [comment]
Seems so. Please consult the manual for the specific order which should be used, i.e. "_In all such places you
should use a hexadecimal 0xRRGGBB notation._" Fixing this would be rather low priority, as this is not visible outside the profiler internals. What is your use case?

**Comment by s9w** [comment]
Not sure we're talking about the same thing. I mean the actual color of the zones in tracy itself. I'm using the C API if that matters. My `tracy::Color::Red` zone is blue.

**Comment by wolfpld** [comment]
Ok, this is indeed happening. Also, it seems that `ZoneScopedC()` is somehow lost, only using `ZoneScoped` + `ZoneColor()` works. Will investigate later.

**Comment by wolfpld** [comment]
> Also, it seems that `ZoneScopedC()` is somehow lost, only using `ZoneScoped` + `ZoneColor()` works.

This was caused by using on-demand mode.

**Comment by wolfpld** [comment]
Should be fixed in 6d571090.
